### DISCUSSION TITLE
Different sampling-rate of service instances in the calling of RPC.

### DISCUSSION THREAD

**Discussion by Fang-Bw** [apache/skywalking#10824] [Q&A] [answered]
For RPC framework like Dubbo, if there is a calling "A->B->C".
Each service set different sampling-rate with agent.config(or even same sampling-rate), how to ensure there is always a complete trace of "A->B->C", which means each trace of this calling can always contains A, B and C.

The reason of this question is that I want to lower the sampling cost of service instance with low sampling-rate as possible.

**Comment by wu-sheng** [comment, marked as answer]
If you set sampling policy at agent side only, the trace is alwayd continued if the previous node is sampled.
